tests/test-cases/Group23-VIC-Machine-Service/23-03-VCH-Create.robot

```diff
@@ -54,10 +61,36 @@ Inspect VCH Config ${name}
 Get VCH ${name}
     Get Path Under Target    vch
     ${id}=    Run    echo '${OUTPUT}' | jq -r '.vchs[] | select(.name=="${name}").id'
+    Set Test Variable    ${id}
 
     Get Path Under Target    vch/${id}
 
 
+Pull Busy Box
+    [Arguments]    ${docker_params}    ${expected_rc}    ${expected_output}
+
+    ${rc}  ${output}=  Run And Return Rc And Output  docker -H ${docker_params} pull busybox
+
+    Should Be Equal As Integers    ${rc}    ${expected_rc}
+    Should Contain    ${output}    ${expected_output}
+
+
+Get Docker Host Params
+    [Arguments]    ${vch}    ${vch_name}
+    Run Keyword And Continue On Failure    Wait Until Keyword Succeeds    20x    10s    Get Docker Params API    ${vch}    ${vch_name}
+    Run Keyword If    "${docker_host}" == "null"    Run VIC Machine Inspect Command   ${vch_name}
```

Comment:
Why fall back to the CLI here? Are you seeing cases where the API doesn't accurately report this information?

If we can't reliably get this information from the API, that seems like a serious bug that should cause the test to fail. (And not falling back to the CLI seems like it would simplify the logic here.)

Reply:
Increasing it to 5min @zjs .